release/Makefile

```diff
@@ -14,13 +14,15 @@ windows:
 copy:
 	rm -rf installers
 	mkdir installers
-	cp macos/build/*.pkg installers
-	cp windows/build/trezor-bridge*.exe installers
+	cp macos/build/*.pkg macos/build/*.pkg.asc installers
+	cp windows/build/trezor-bridge*.exe windows/build/trezor-bridge*.exe.asc installers
 	cp linux/build/*.deb linux/build/*.rpm installers
-
-gpg:
-	gpg -u 54067D8BBF00554181B5AB8F26A3A56662F0E7E2 --armor --detach-sig installers/trezor-bridge*.exe
-	gpg -u 54067D8BBF00554181B5AB8F26A3A56662F0E7E2 --armor --detach-sig installers/*.pkg
+	rm -rf binaries
+	mkdir binaries
+	cp macos/build/trezord binaries/trezord-darwin-universal
```

Comment:
I have added new folder "binaries" for all the built binaries. (Different than the installers.)